vello/src/lib.rs

```diff
@@ -134,14 +134,25 @@ use wgpu::{Device, Queue, SurfaceTexture, TextureFormat, TextureView};
 use wgpu_profiler::{GpuProfiler, GpuProfilerSettings};
 
 /// Represents the antialiasing method to use during a render pass.
-#[derive(Copy, Clone, PartialEq, Eq)]
+#[derive(Debug, Copy, Clone, PartialEq, Eq)]
 pub enum AaConfig {
     Area,
     Msaa8,
     Msaa16,
 }
 
+impl From<AaConfig> for AaSupport {
+    fn from(value: AaConfig) -> Self {
+        Self {
+            area: value == AaConfig::Area,
+            msaa8: value == AaConfig::Msaa8,
+            msaa16: value == AaConfig::Msaa16,
+        }
+    }
+}
+
```

Comment:
We should either:
```suggestion
```
or implement [`BitOr`](https://doc.rust-lang.org/std/ops/trait.BitOr.html) for `AaSupport` if we're doing this.

Reply:
Yeah, removing makes sense now that I know how the support/config work.